gcloud/Gemfile

```diff
@@ -2,38 +2,36 @@ source "https://rubygems.org"
 
 gemspec
 
-gem "rake", "~> 11.0"
+gem "gcloud-jsondoc",
+    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
+    branch: "gcloud-jsondoc"
 gem "google-cloud", path: "../google-cloud"
-gem "google-cloud-core", path: "../google-cloud-core"
-gem "google-cloud-env", path: "../google-cloud-env"
 gem "google-cloud-bigquery", path: "../google-cloud-bigquery"
+gem "google-cloud-container", path: "../google-cloud-container"
```

Comment:
Was google-cloud-container added here? It seems this should just be an alphabetical sort. If it was added, why? (Sorry if explained in a commit msg, I'm looking at all commits together.)

Reply:
It is added to `google-cloud`, and because of that `gcloud` need to know where to find it. Otherwise it may try to download the gem from rubygems instead of getting it locally.

Reply:
The reason I was poking at the Gemfile at all is because newer Rubocop has a rule to list the gems in the Gemfile alphabetically, so I updated all the Gemfiles and made sure they were correct. This change was to correct the missing dependency.

Reply:
It seems to me that updating dependencies would be better in separate PR, but OK, I understand now.